elasticsearch/utils/es_util_env

```diff
@@ -6,3 +6,18 @@ fi
 
 ES_BASE=${ES_BASE:-https://localhost:9200}
 curl_get='curl -s -X GET --cacert /etc/elasticsearch/secret/admin-ca --cert /etc/elasticsearch/secret/admin-cert --key /etc/elasticsearch/secret/admin-key'
+
+
+get_security_conf_index() {
```

Comment:
this function is here and also in `utils/logging` ?

Reply:
yes - so - I don't know if it is safe to source `utils/logging` from scripts that use `es_util_env` and vice versa - looks like an opportunity for some refactoring here